build/release-notes.md

```diff
@@ -0,0 +1,524 @@
+---
+title: Build release notes
+description: Release notes for Buildx
+keywords: build, buildx, buildkit, release notes
+toc_max: 2
+---
+
+This page contains information about the new features, improvements and bug
+fixes in [Buildx](buildx/index.md).
+
+## 0.9.1
+
+{% include release-date.html date="2022-08-18" %} 
+
+### Bug fixes
+
+* Fix regression on building compose files that contain services without a
+  build block {% include github_issue.md repo="docker/buildx" number="1277" %}
+
+### Enhancements
+
+* Ensure used buildkit version also shows up in the inspect command {% include github_issue.md repo="docker/buildx" number="1279" %}
+
+See [full release notes](https://github.com/docker/buildx/releases/tag/v0.9.1){:target="_blank" rel="noopener" class="_"} for more details.
+
+## 0.9.0
+
+{% include release-date.html date="2022-08-17" %} 
+
+### New features
+
+* Support for new [driver `remote`](buildx/drivers/remote.md) that can be used
+  to connect to any already running BuildKit instance {% include github_issue.md repo="docker/buildx" number="1078" %}
+  {% include github_issue.md repo="docker/buildx" number="1093" %} {% include github_issue.md repo="docker/buildx" number="1094" %}
+  {% include github_issue.md repo="docker/buildx" number="1103" %} {% include github_issue.md repo="docker/buildx" number="1134" %}
+  {% include github_issue.md repo="docker/buildx" number="1204" %}
+* Dockerfile can now be loaded from standard input even when build context is
+  coming from external Git or HTTP URL {% include github_issue.md repo="docker/buildx" number="994" %}
+* Build commands now support new build context type `oci-layout://` for loading
+  [build context from local OCI layout directories](../engine/reference/commandline/buildx_build.md#-source-image-from-oci-layout-directory).
+  Note that this feature depends on an unreleased BuildKit feature and builder
+  instance from `moby/buildkit:master` needs to be used until BuildKit v0.11 is
+  released {% include github_issue.md repo="docker/buildx" number="1173" %}
+* New `--print` flag can be used to run helper functions supported by the
+  BuildKit frontend performing the build and print their results. In Dockerfile
+  this feature can be used to show the build arguments and secrets that the
+  current build supports with `--print=outline` and list all available
+  Dockerfile stages with `--print=targets`. This feature is experimental for
+  gathering early feedback and requires enabling `BUILDX_EXPERIMENTAL=1`
+  environment variable. We plan to update/extend this feature in the future
+  without keeping backward compatibility {% include github_issue.md repo="docker/buildx" number="1100" %}
+  {% include github_issue.md repo="docker/buildx" number="1272" %}
```

Comment:
Not sure the github issue include is easier to use than just creating a markdown link🤔 

Was this to avoid having to add the "rel=noopener" properties etc?

Reply:
Just to have something less verbose. Fine to me to switch to pure markdown if you want.